ui/src/app/settings/selectors/config.js

```diff
@@ -332,12 +332,51 @@ config.discoveryIntervalOptions = createSelector(
 
 /**
  * Returns the MAAS config for kernel parameters.
- * @param {object} state - The redux state.
+ * @param {Object} state - The redux state.
  * @returns {String} Kernel parameters.
  */
 config.kernelParams = createSelector(
   [config.all],
   configs => getValueFromName(configs, "kernel_opts")
 );
 
+/**
+ * Returns the MAAS config for default OS.
+ * @param {Object} state - The redux state.
+ * @returns {String} Default OS.
+ */
+config.defaultOSystem = createSelector(
+  [config.all],
+  configs => getValueFromName(configs, "default_osystem")
+);
+
+/**
+ * Returns the possible default OS options reformatted as objects.
+ * @param {Object} state - The redux state.
+ * @returns {Array} Default OS options.
+ */
+config.defaultOSystemOptions = createSelector(
+  [config.all],
+  configs => {
+    const configObj = configs.find(config => config.name === "default_osystem");
```

Comment:
There's a helper function `getOptionsFromName` above that you can use instead

Reply:
done